src/features/embalm/stepNavigator/hooks/useSetStatuses.ts

```diff
@@ -2,13 +2,23 @@ import { useEffect } from 'react';
 import { updateStepStatus } from 'store/embalm/actions';
 import { Step, StepStatus } from 'store/embalm/reducer';
 import { useDispatch, useSelector } from 'store/index';
+import { ethers } from 'ethers';
+
+export function validatePublicKey(key: string) {
```

Comment:
Right here is a fine place for this I think. If we need to add more validation functions I would create a separate folder for them.